docs/src/testing/coverage.md

```diff
@@ -5,6 +5,8 @@ With `cairo-coverage`, you can generate a coverage report that can later be anal
 
 ## Prerequisites
 
+`cairo-coverage` relies on debug information provided by Scarb. To generate the necessary debug information, you need to have:
```

Comment:
```suggestion
`cairo-coverage` relies on debug information provided by Scarb. To generate the necessary debug information, you need to install:
```

Reply:
`install` might not be the most accurate in this context given that the 2. point is about configuring `Scarb.toml`